File: spring-boot-VMM/src/main/java/pp/project/vmm/config/security/rest/request/RegisterRequest.java
package pp.project.vmm.config.security.rest.request;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;
import java.util.Set;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class RegisterRequest {

    private String username;
    private String password;
    private String firstName;
    private String lastName;
    private String phoneNumber;
    private List<String> roles;
}

File: angular-VMM/src/app/models/Employee/RegisterRequest.ts
export class RegisterRequest {
    username: string;
    password: string;
    firstName: string;
    lastName: string;
    phoneNumber: string;
    roles: string[];


    constructor(username: string, password: string, firstName: string, lastName: string, phoneNumber: string, roles: string[]) {
        this.username = username;
        this.password = password;
        this.firstName = firstName;
        this.lastName = lastName;
        this.phoneNumber = phoneNumber;
        this.roles = roles;
    }
}
